### ISSUE TITLE
Determine required software, items, etc.

### ISSUE THREAD

**Issue by nicholaswharris** [nicholaswharris/swe_proposal#2]
Anything missing?

Linux, Windows, or Mac (OS)
CUDA-Capable GPU (**highly recommended**)
Pip, Conda, or LibTorch (package manager)
Python 3.x or C++/Java (programming language)
Pytorch (Machine Learning library)
Plotly and/or Pandas (libraries for graphing, analytics, statistics)
Dataset (pictures)
Templates (Neural Network models)

**Comment by jtcooper10** [comment]
Might be worth considering adding Plotly and/or Pandas for plotting things like accuracy, as well.

Also templates? Not sure where to even find those. Aside from that, looks good to me.
